dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.javax.annotation-1.1.feature

```diff
@@ -4,6 +4,7 @@ WLP-DisableAllFeatures-OnConflict: false
 singleton=true
 IBM-Process-Types: server, \
  client
+-features=com.ibm.websphere.appserver.eeCompatible-6.0; ibm.tolerates:="7.0, 8.0"
 -bundles=com.ibm.websphere.javaee.annotation.1.1; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.annotation:javax.annotation-api:1.2"
```

Comment:
This is not from your change, but the maven coordinates seem wrong.

Reply:
> This is not from your change, but the maven coordinates seem wrong.

I agree, but don't plan to address in this PR